Documentation/media-migration.md

```diff
@@ -1,220 +1,828 @@
+
+playheadUpdatePingXDM = {
+  "xdm": {
+    "eventType": "media.ping",
+    "mediaCollection": {
+      "playhead": <CURRENT_PLAYHEAD_INTEGER_VALUE>
+    }
+  }
+}
+
+``` brightscript
+m.aepSdk.sendMediaEvent(playheadUpdatePingXDM)
+```
+
+---
+
 ### updateQoEObject
 
-- Media SDK
+**Media SDK**
 
 ``` brightscript
-m.adbmobile.mediaUpdateQoS
+' Use the helper method to create mediaInfo object
+bitrate = 200000
+fps = 24
+droppedFrames = 1
+startupTime = 2
+qosInfo = m.adbmobile.adb_media_init_qosinfo(bitrate, startupTime, fps, droppedFrames)
+
+m.adbmobile.mediaUpdateQoS(qosInfo)
 ```
 
-- AEP SDK
+**AEP SDK**
+
+> [!IMPORTANT]
+> All the QoE field values should be of `Integer` data type.
 
 ``` brightscript
+qoeDataDetails = {
+  "bitrate" : 200000,
+  "framesPerSecond" : 24,
+  "droppedFrames" : 1,
+  "timeToStart" : 2
+}
+
+bitrateChangeXDM = {
+  "xdm": {
+    "eventType": "media.bitrateChange",
+    "mediaCollection": {
+      "playhead": <CURRENT_PLAYHEAD_INTEGER_VALUE>,
+      "qoeDataDetails": qoeDataDetails
+    }
+  }
+}
+
+m.aepSdk.sendMediaEvent(bitrateChangeXDM)
+```
+
+> [!NOTE]
+> To learn more refer to the [qoeDataDetails](https://github.com/adobe/xdm/blob/master/docs/reference/datatypes/qoedatadetails.schema.md) XDM fieldgroup.
+> `qoeDataDetails` is supported and can be attached to any event going out.
+
+Following is the sample to send qoeDataDetails with play event:
+
+``` brightscript
+playWithQoeXDM = {
+  "xdm": {
+    "eventType": "media.play",
+    "mediaCollection": {
+      "playhead": <CURRENT_PLAYHEAD_INTEGER_VALUE>,
+      "qoeDataDetails": qoeDataDetails
+    }
+  }
+}
+
+m.aepSdk.sendMediaEvent(playWithQoeXDM)
+```
+---
+
+### trackError
+
+**Media SDK**
+
+``` brightscript
+m.adbmobile.mediaTrackError("errorId", "video-player-error-code")
 ```
+
+**AEP SDK**
+``` brightscript
+errorDetails = {
+  "name": "errorId",
+  "source": "video-player-error-code"
+}
+
+errorXDM = {
+  "xdm": {
+    "eventType": "media.error",
+    "mediaCollection": {
+      "playhead": <CURRENT_PLAYHEAD_INTEGER_VALUE>,
+      "errorDetails": errorDetails
+    }
+  }
+}
+
+m.aepSdk.sendMediaEvent(errorXDM)
+```
+
+> [!NOTE]
+> To learn more refer to the [errorDetails](https://github.com/adobe/xdm/blob/master/docs/reference/datatypes/errordetails.schema.md) XDM fieldgroup.
+
+## Public Constants
+
+### StreamType
+
+| Media SDK | AEP SDK |
+| -- | -- |
+| MEDIA_STREAM_TYPE_VOD | NA |
+| MEDIA_STREAM_TYPE_LIVE | NA |
+| MEDIA_STREAM_TYPE_LINEAR | NA |
+| MEDIA_STREAM_TYPE_AOD | NA |
+| MEDIA_STREAM_TYPE_AUDIOBOOK | NA |
+| MEDIA_STREAM_TYPE_PODCAST | NA |
+
+### MediaType
+
+| Media SDK | AEP SDK |
+| -- | -- |
+| MEDIA_STREAM_TYPE_AUDIO | NA |
+| MEDIA_STREAM_TYPE_VIDEO | NA |
+
+### Standard Video Metadata
+
+| Media SDK | AEP SDK |
+| -- | -- |
+| MEDIA_VideoMetadataKeySHOW                    | Refer to [sessionDetails](https://github.com/adobe/xdm/blob/master/docs/reference/datatypes/sessiondetails.schema.md) XDM fieldgroup |
```

Comment:
Can you provide the link to the correct field?
https://github.com/adobe/xdm/blob/master/docs/reference/datatypes/sessiondetails.schema.md#xdmshow